packages/fxa-settings/src/components/PageRecoveryKeyAdd/index.tsx

```diff
@@ -80,6 +85,7 @@ export const PageRecoveryKeyAdd = (_: RouteComponentProps) => {
     },
   });
   useEffect(() => {
+    // TODO what's this?
```

Comment:
this is for if you arrive at this page with an invalid state like by doing a hard refresh on step 2